CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning (http://semver.org/).
 
+##  4.0.0
+* Handle order operation through GraphQL and remove APIs
```

Comment:
"Handle order operation through GraphQL and remove APIs" -> "Handle order confirmations through GraphQL instead of legacy REST API"

Also add line: "Introduce operation for sending order status updates separately "